examples/itk-vtk-viewer/createTransferFunctionEditor.js

```diff
@@ -2,7 +2,7 @@ import { throttle } from '@kitware/vtk.js/macros'
 import {
   TransferFunctionEditor,
   windowPointsForSort,
-} from '../../lib/TransferFunctionEditor'
+} from 'itk-viewer-transfer-function-editor'
```

Comment:
My understanding of current path best practices is to use the `.js` extension, i.e. `itk-viewer-transfer-function-editor.js`, because the ESM / browser use case does not support discovery of file extensions.

Reply:
Interesting ✅